src/Matrix.jl

```diff
@@ -3558,6 +3558,8 @@ If `side` is `:right` and `flag` is true, $B$ is the right inverse of $A$
 i.e. $AB$ is the $n\times n$ unit matrix. If `side` is `:left` and `flag` is
 true, $B$ is the left inverse of $A$ i.e. $BA$ is the $m\times m$ unit matrix.
 If `flag` is false, no right or left inverse exists.
+
+To get the space of all inverses, combine with [`nullspace`](@ref).
```

Comment:
```suggestion
To get the space of all inverses, combine with [`kernel`](@ref).
```

Since the linear algebra cleanup we are promoting `kernel` over `nullspace`.

Reply:
Thanks!